.gitignore

```diff
@@ -28,3 +28,7 @@ package.json.ember-try
 
 .DS_Store
 yarn-error.log
+
+#Eclipse
```

Comment:
This change isn't necessary. If it's the editor you use, then it should be in your global gitignore not here.